package-lock.json

```diff
@@ -10437,9 +10437,9 @@
       }
     },
     "csstype": {
-      "version": "2.6.3",
```

Comment:
Why did we have csstype in package-lock if we are only adding it now?

Reply:
it's a dependency of `@emotion`!

Reply:
That makes sense, I guess.

Reply:
here's a boiled down section of the `package-lock.json` that hopefully makes some sense:

```
```